GitUI/CommandsDialogs/FormCheckoutBranch.cs

```diff
@@ -168,45 +168,26 @@ private void ApplyLayout()
                 localChangesPanel
             };
 
-            if (EnvUtils.IsMonoRuntime())
-            {
-                var margin = flowLayoutPanel2.Controls[0].Margin;
-                flowLayoutPanel2.Height = flowLayoutPanel2.Controls.OfType<Control>().Max(c => c.Height) + margin.Top + margin.Bottom;
-                localChangesGB.Height = flowLayoutPanel2.Height * 2 + localChangesGB.Padding.Top + localChangesGB.Padding.Bottom;
-                localChangesPanel.Height = localChangesGB.Height + localChangesGB.Margin.Top + localChangesGB.Margin.Bottom;
-
-                Ok.AutoSize = false;
-                var buttonWidth = TextRenderer.MeasureText(Ok.Text, AppSettings.Font).Width + 10;
+            Ok.Anchor = AnchorStyles.Right;
 
-                localChangesPanel.ColumnStyles[1].Width = buttonWidth + Ok.Margin.Left + Ok.Margin.Right;
-                Ok.Width = buttonWidth;
+            flowLayoutPanel2.AutoSize = true;
+            flowLayoutPanel2.Dock = DockStyle.Fill;
+            localChangesGB.AutoSize = true;
+            localChangesGB.Height = flowLayoutPanel2.Height * 2 + localChangesGB.Padding.Top + localChangesGB.Padding.Bottom;
+            localChangesPanel.ColumnStyles[1].Width = Ok.Width + 10;
+            var height = localChangesGB.Height + localChangesGB.Margin.Top + localChangesGB.Margin.Bottom;
+            localChangesPanel.RowStyles[0].Height = height;
+            localChangesPanel.Height = height;
 
-                localChangesGB.Width = localChangesPanel.Width - (int)localChangesPanel.ColumnStyles[1].Width;
-                flowLayoutPanel2.Width = localChangesGB.Width - 2 * localChangesGB.Padding.Left - 2 * localChangesGB.Padding.Right;
-            }
-            else
-            {
-                Ok.Anchor = AnchorStyles.Right;
-
-                flowLayoutPanel2.AutoSize = true;
-                flowLayoutPanel2.Dock = DockStyle.Fill;
-                localChangesGB.AutoSize = true;
-                localChangesGB.Height = flowLayoutPanel2.Height * 2 + localChangesGB.Padding.Top + localChangesGB.Padding.Bottom;
-                localChangesPanel.ColumnStyles[1].Width = Ok.Width + 10;
-                var height = localChangesGB.Height + localChangesGB.Margin.Top + localChangesGB.Margin.Bottom;
-                localChangesPanel.RowStyles[0].Height = height;
-                localChangesPanel.Height = height;
-
-                Width = tableLayoutPanel1.PreferredSize.Width + 40;
-            }
+            Width = tableLayoutPanel1.PreferredSize.Width + 40;
 
             for (var i = 0; i < controls1.Length; i++)
             {
                 var margin = controls1[i].Margin;
-                var height = controls1[i].Height + margin.Top + margin.Bottom;
-                _controls.Add(controls1[i], height);
+                var cheight = controls1[i].Height + margin.Top + margin.Bottom;
```

Comment:
Is there any reason we couldn't reuse `height` var?

Reply:
The declaration of 'height' prior in the function is no longer in a separate block.
I would prefer the blocks to be independent still, but in this case it makes sense to reuse 'height'